app/views/pages/cookies.html.erb

```diff
@@ -3,23 +3,53 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds-from-desktop">
     <h1 class="govuk-heading-l govuk-!-margin-bottom-9">Cookies on <%= t('service_name') %></h1>
-    
+
     <p class="govuk-body">This service puts small files (known as ‘cookies’) onto your computer, phone or tablet to collect information about how you use the service.</p>
-    
+
     <h2 class="govuk-heading-m">What cookies we use</h2>
-    
+
     <p class="govuk-body">We only use essential cookies.</p>
-    
+
     <h3 class="govuk-heading-s">A cookie to give you access to your account</h3>
-    
+
     <p class="govuk-body">"Register_trainee_teachers_session" is a cookie that reminds the service who you are when you’re signed in.</p>
-    
+
     <p class="govuk-body">It enables you to access the information in your account, such as your trainee records.</p>
-    
+
     <p class="govuk-body">It expires when you sign out, or after 6 hours on the service.</p>
 
     <h3 class="govuk-heading-s">A cookie to store that you have seen the cookies banner.</h3>
 
-<p class="govuk-body">"viewed_cookie_message" is a cookie which tells the service that you have seen the cookies banner so we do not show it to you again.</p>
+    <p class="govuk-body">"viewed_cookie_message" is a cookie which tells the service that you have seen the cookies banner so we do not show it to you again.</p>
+
+    <h3 class="govuk-heading-s">Measuring website usage (Google Analytics)</h3>
+    <p class="govuk-body">Google Analytics sets the following cookies:</p>
```

Comment:
Should we have something saying why we use google analytics?

Eg, GOV.UK has this to say:
![Screenshot 2021-03-08 at 17 25 15](https://user-images.githubusercontent.com/2204224/110357611-5ef90e80-8033-11eb-863c-d9cc5212045c.png)

Reply:
Sorted